.github/ISSUE_TEMPLATE/DOCUMENTATION.yaml

```diff
@@ -0,0 +1,44 @@
+name: Documentation
+description: Report typos/bugs or suggest new content.
+labels: ["documentation"]
+assignees: 'abbycross, beckykd, javabster'
```

Comment:
I would imagine (but am not sure) that this should be
```yaml
- assigness:
  - abbycross
  - beckykd
  - javabster
```
and that the current form would look for a single username that doesn't exist.

Reply:
Ah you're likely right -- I copied over from an md template.